docs/admin/optionals.rst

```diff
@@ -86,22 +81,13 @@ should run the database migration:
 Usage
 +++++
 
-After installation you can control billing in the admin interface. Users with
-billing enabled will get new :guilabel:`Billing` tab in their
-:ref:`user-profile`.
-
-The billing module additionally allows project admins to create new projects
-and components without being superusers (see :ref:`adding-projects`). This is
-possible when following conditions are met:
+Installation lets you control billing and create/import new components and projects from your admin interface
+if the billing is within its configured limits (any overusage prevents project/component creation)
+and paid (if its price is non zero).
 
-* The billing is in it's configured limits (any overusage results in blocking
-  of project/component creation) and paid (if it's price is non zero)
-* The user is admin of existing project with billing or user is owner of
-  billing (the latter is necessary when creating new billing for users to be
-  able to import new projects).
+Admins can create new components for their projects without escalating to superuser status (see :ref:`adding-projects`).
 
-Upon project creation user is able to choose which billing should be charged
-for the project in case he has access to more of them.
+:ref:`integrating-support` lets users in charge of billing create new projects from the :guilabel:`Billing` tab in their :ref:`user-profile`. A choice for which billing account to charged will be presented if more than one is available.
```

Comment:
Not sure where this should reference to...

Reply:
@comradekingu can you check, please?

Reply:
Which one? I don't really know myself.

Reply:
I think
https://docs.weblate.org/en/latest/admin/support.html#integrating-support https://docs.weblate.org/en/latest/admin/optionals.html#billing and
https://docs.weblate.org/en/latest/user/profile.html#user-profile

Reply:
The billing module is independent of purchasing support. It can be also used on self-hosted instances to enforce usage limits (for example number of projects user can create).